CONTRIBUTING.md

```diff
@@ -0,0 +1,84 @@
+# Contributing to ISOBUS++
+
+We accept public contributions that follow our code of conduct, pass all automated pre-merge checks, and pass a manual code review by a repo maintainer.
+
+## What Are The Requirements?
+
+Our style rules and PR reviews are based loosely on Autosar and MISRA suggestions and try to ensure the highest quality possible.
+
+* Contributions must follow the style defined in our `.clang-format` file. You can ensure you pass this check by running `find . -iname *.hpp -o -iname *.cpp | xargs clang-format -i` at the root of the repo.
+* Contributions should follow these additional style requirements, which will be checked in code reviews.
+	* Function names `snake_case`
+	* Variables `camelCase`
+	* Constant values `CAPITALIZED_SNAKE`
+	* Constants on the left in `==` and `!=` checks. Like this: `if (5 == value)` NOT `if (value == 5)`. This is to prevent accidentally omitting an `=` in the operator and creating a runtime bug.
```

Comment:
Could it be useful to add a line about usage of `std::` here?

Reply:
Furthermore, this should really be a given for any C++ developer, but a note that `using namespace x` should never be in a header file may be good to have